wellcomecollection/wellcomecollection.org
# Add prismic linting step for audio/video duration format (xx:xx)

## What does this change?
We've changed the Guide Stop audio/video duration fields to be strings rather than numbers, and we're requesting that editors enter them in the format xx:xx but we can't validate this at the point of data entry (because Prismic) so this adds a rule to the `lintPrimsic` script to check the duration strings against a regex that expects {number}{number}:{number}{number}. 

## How to test
I deliberately entered the last of the video durations for [Jason](https://wellcomecollection.org/guides/exhibitions/ZthrZRIAACQALvCC/bsl) as `4:37` (instead of `04:37`) to verify that we're catching the errors. I'm planning on updating this, but can wait if you'd also like to test it yourselves.

## How can we measure success?
Site has consistent guide-stop durations

## Have we considered potential risks?
~A guide stop with a/v durations longer than an hour would cause issues. Might be worth making the regex a bit more tolerant?~ <– did this
